### PR TITLE
ADD Jean-Baptiste Beuzelin

### PR DESCRIPTION
**Pull request by jbbeuzelin**

### CODE REVIEW

File: baggers.js [hunk 1]
```diff
@@ -13899,6 +13899,45 @@ var data = {
             "Paris",
             "Lyon"
         ]
+    },
+    {
+      "since": "2016-11-10",
```
Comment (dadoonet) [inline baggers.js]: "2016-11-20"?

File: baggers.js [hunk 2]
```diff
@@ -13899,6 +13899,45 @@ var data = {
...
+      "name": "Jean-Baptiste Beuzelin",
+      "bio": "Ingénieur R&D chez Lucca, je m'occupe de développer une solution interne de gestion de parc client et de facturation.",
+      "picture": "http://www.lucca.fr/img/equipe/jbeuzelin.png",
+      "websites": [
+        {
+          "name": "Lucca",
```
Comment (dadoonet) [inline baggers.js]: Please use one of the name described in doc: https://github.com/brownbaglunch/bblfr_data#websites. I suggest here: `Web`

**Review by dadoonet — CHANGES_REQUESTED** [review]

**Comment by jbbeuzelin** [comment]
Fixs committed @dadoonet . Thx

**Comment by jbbeuzelin** [comment]
@dadoonet I connected my brain and so I made a real fix this time. Sorry

**Comment by dadoonet** [comment]
Bienvenue !

**Pull request merged**